docs/www/autotune.md

```diff
@@ -83,3 +83,4 @@ Swapping is prevented so that Redpanda is never swapped out of memory. By
 design, `redpanda` allocates nearly all of the available memory upfront,
 partitioning the allocated memory between all cores and pinning such memory
 to the specified NUMA domain (specific CPU socket). This makes sure that we have predictable memory allocations and provides predictable latency.
+We recommend having at least 1GB of memory.
```

Comment:
I think check was enforcing 1 GB per CPU not 1 GB in total. I'll defer to others here, but my guess is that it's best to simply not mention this here.

Reply:
Done.